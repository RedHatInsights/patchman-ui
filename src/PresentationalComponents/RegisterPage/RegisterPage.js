import { Main } from '@redhat-cloud-services/frontend-components';
import React from 'react';
import Header from '../Header/Header';
import { NoSystemData } from '../Snippets/NoSystemData';
import { intl } from '../../Utilities/IntlProvider';
import messages from '../../Messages';

const RegisterPage = () => {
    return (
        <React.Fragment>
            <Header title={intl.formatMessage(messages.generalAppName)}/>
            <Main>
                <div>
                    <NoSystemData/>
                </div>
            </Main>
        </React.Fragment>
    );
};

export default RegisterPage;
