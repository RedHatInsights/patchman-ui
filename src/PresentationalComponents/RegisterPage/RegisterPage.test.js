import RegisterPage from './RegisterPage';
import toJson from 'enzyme-to-json';

describe('RegisterPage', () => {
    it('Should match the snapshot', () => {
        const wrapper = shallow(<RegisterPage />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
