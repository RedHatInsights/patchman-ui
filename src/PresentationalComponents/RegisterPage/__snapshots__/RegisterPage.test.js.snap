// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegisterPage Should match the snapshot 1`] = `
<Fragment>
  <Header
    title="Patch"
  />
  <Connect(Main)>
    <div>
      <NoSystemData />
    </div>
  </Connect(Main)>
</Fragment>
`;
